logging/logger_registry.go

```diff
@@ -149,6 +149,14 @@ func LoggerNamed(name string) (logger Logger, ok bool) {
 	return globalLoggerRegistry.loggerNamed(name)
 }
 
+func GetOrNewLogger(name string) (logger Logger) {
+	logger, ok := globalLoggerRegistry.loggerNamed(name)
+	if !ok {
+		return NewLogger(name)
+	}
+	return logger
+}
```

Comment:
I don't think there is a clear natural place to initialize a e.g. `rdk.network_outgoing` logger so this helper lets us safely retrieve and use the same logger object globally without fear of overwriting a map entry to a new logger with the same name.